tests/torch/test_models/synthetic.py

```diff
@@ -148,6 +148,39 @@ def forward(self, x):
         return xa, xb, xc, xd
 
 
+class PartlyNonDifferentialOutputsModel(nn.Module):
+    def __init__(self, input_size=None):
+        super().__init__()
+        self.input_size = [1, 1, 4, 4] if input_size is None else input_size
+        self.Conv1 = torch.nn.Conv2d(in_channels=self.input_size[1], out_channels=1, kernel_size=3)
+        self.Conv2_1 = torch.nn.Conv2d(in_channels=1, out_channels=1, kernel_size=3)
+        self.Conv2_2 = torch.nn.Conv2d(in_channels=1, out_channels=1, kernel_size=3)
```

Comment:
Variables in Python should be lower-cased, you can see that something is wrong already in the Github's code highlighter.